### PR TITLE
feat(cli): apiv2 support for query run

### PR DESCRIPTION
**Pull request by hazedav**
ExecuteByID is now restful
Eliminated InputQueryFromEnv
Simplified --validate_only

ALLY-562

### CODE REVIEW

File: api/lql_execute.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+import (
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	RFC3339Milli = "2006-01-02T15:04:05.000Z"
```
Comment (afiune) [inline api/lql_execute.go:30]: Could we define this time type inside our `lwtime` package?

File: cli/cmd/lql.go [hunk 2]
```diff
@@ -61,8 +61,9 @@ NOTE: This feature is not yet available!`,
 
 	// queryRunCmd represents the lql run command
 	queryRunCmd = &cobra.Command{
-		Use:   "run [query_id]",
-		Short: "run a query",
+		Aliases: []string{"execute"},
+		Use:     "run [query_id]",
```
Comment (afiune) [inline cli/cmd/lql.go:65]: The standard for arguments is `<arg>` not `[arg]`:
```suggestion
		Use:     "run <query_id>",
```

**Review by afiune — APPROVED** [review]
A couple of things that need to change, out of that, this change looks good! 💯

File: api/lql_execute.go (hunk 1 above)
Comment (hazedav) [reply api/lql_execute.go:30]: Yes; absolutely...

File: cli/cmd/lql.go (hunk 2 above)
Comment (hazedav) [reply cli/cmd/lql.go:65]: I'm pretty sure you told me that optional arguments use square brackets...

**Pull request merged**